.gitignore

```diff
@@ -2,3 +2,4 @@
 /lake-packages
 /.cache
 .DS_Store
+/lakefile.olean
```

Comment:
Should we just ignore `*.olean` in general?

Reply:
Other oleans are, for now, a sign of something being wrong, so I'd prefer not to ignore them.